apps/qubit/modules/contactinformation/actions/indexAction.class.php

```diff
@@ -108,6 +108,30 @@ public function execute($request)
       $value['contactType'] = $this->resource->contactType;
     }
 
+    // Add source culture information if necessary
+    $culture = sfContext::getInstance()->user->getCulture();
+
+    if ($culture != $this->resource->sourceCulture)
+    {
+      $sourceCultureData = array('fields' => array());
+
+      // Add source culture direction, if necessary
+      $sourceCultureInfo = sfCultureInfo::getInstance($this->resource->sourceCulture);
```

Comment:
Is this really necessary, why can we get the source culture value in here like it's being done bellow? Could you explain a little why is this needed in a comment?

Reply:
It makes the below line below 40 characters shorter, but I'll get rid of it as it does make the code a bit harder to follow.

Reply:
Oh, I reference the culture info on line 122 as well... I've changed it so I just take store the source culture direction in a convenience variable, rather than general source culture info, to make the purpose of the convenience variable more explicit.

Reply:
(I've expanded the comment a bit to clarify function of the direction data a bit. Thanks!)